api/src/main/resources/liquibase.xml

```diff
@@ -347,5 +347,14 @@
         </createIndex>
     </changeSet>
 
+    <changeSet id="add_cohort_foreignKeyConstraint" author="manojll">
```

Comment:
Can we change the id such that when we later on add a foreign key to another field, we still find it meaningful?

Reply:
@dkayiwa updated